pythonforandroid/bootstraps/webview/__init__.py

```diff
@@ -7,7 +7,9 @@
 class WebViewBootstrap(Bootstrap):
     name = 'webview'
 
-    recipe_depends = ['genericndkbuild', ('python2', 'python3', 'python3crystax')]
+    recipe_depends = list(
+        set(Bootstrap.recipe_depends).union(set(['genericndkbuild']))
```

Comment:
won't it be a little bit more concise/Pythonic to use literals and operators?
Something like:
```python
recipe_depends = list(set(Bootstrap.recipe_depends) | {'genericndkbuild'})
```

Reply:
I haven't checked if `|` is recommended over the `union`, it was just a matter of taste.
So feel free to use `union` if you prefer until I prove PEP8 says otherwise :wink: